libs/coin-modules/coin-aptos/src/logic/getBalance.ts

```diff
@@ -0,0 +1,15 @@
+import type { Balance } from "@ledgerhq/coin-framework/lib/api/types";
+import type { AptosAsset } from "../types/assets";
+import type { AptosAPI } from "../network";
+
+export async function getBalance(
+  aptosClient: AptosAPI,
+  address: string,
+): Promise<Balance<AptosAsset>[]> {
+  const balance = await aptosClient.getBalances(address);
+
+  return balance.map(x => ({
+    value: BigInt(x.amount.toString()),
+    asset: { type: "native" },
```

Comment:
this is not accurate, we need to known which balance type it is, "native" for native asset, token for FA

Reply:
As we discussed earlier, we'll deal only with native assets in this version.